File: flxxwang_1.0.98_src/source_files/Infomation.Core/InfoCategory/JobCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Infomation.Core
{
    /// <summary>
    /// 职位类别.(求职/招聘时用)
    /// </summary>
    public class JobCategory:Infomation.Core.InfoCategory
    {
        public byte Id
        {
            get;
            set;
        }
        public JobType JobType
        {
            get
            {
                return (JobType)Id;
            }
        }
    }
}

File: flxxwang_1.0.98_src/source_files/Framework.MvcWeb/Content/Fenlei/Scripts/jobCategory.js
//by lls
if (!$info) {
    $info = {}
}
$info.jobCategory = {
    __type: "Infomation.Core.InfoCategory",
    getParents: function (success, error, complete, context) {
        var __jobCategory = new $f.model(this.__type);
        __jobCategory._post("GetParentJobCategories", null, success, error, complete, context);
    },
    getChild: function (parentId, success, error, complete, context) {
        var __jobCategory = new $f.model(this.__type);
        __jobCategory._post("GetChildCategory", [parentId], success, error, complete, context);
            
    }
};